core/components/molecules/modalDescription/__tests__/ModalDescription.test.tsx

```diff
@@ -34,9 +34,13 @@ describe('ModalDescription component with props', () => {
   });
 
   it('ModalDescription with prop: title', () => {
-    const { getByTestId } = render(<ModalDescription title={StringValue} />);
+    const { getAllByTestId } = render(<ModalDescription title={StringValue} />);
 
-    expect(getByTestId('DesignSystem-ModalDescription--Title').textContent).toMatch(StringValue);
+    const titles = getAllByTestId('DesignSystem-ModalDescription--Title');
+
+    for (const title of titles) {
+      expect(title.textContent).toMatch(StringValue);
+    }
```

Comment:
Why do we need to change this test case?

Reply:
On addition of withInput prop with a boolean flag to Label inside ModalDescription component, getByTestId fails to capture both the outputs( one with withInput false, other with withInput true). Hence we need to replace getByTestId with getAllByTestId.